core/templates/pages/exploration-editor-page/services/exploration-category.service.ts

```diff
@@ -32,6 +32,10 @@ angular.module('oppia').factory('ExplorationCategoryService', [
     var child = Object.create(ExplorationPropertyService);
     child.propertyName = 'category';
     child._normalize = $filter('normalizeWhitespace');
+    /**
+     * @param {string} value - The input to be validate.
+     * @returns { boolean } - True if category name is valid.
```

Comment:
```suggestion
     * @returns {boolean} - True if category name is valid.
```

Looking at other files, we don't seem to use extra spaces between `{}` for doc strings.

Reply:
Ditto below.

Reply:
I fixed it. Thank you!